src/plugins/legacy-authorization/util/hasPermission.js

```diff
@@ -6,7 +6,7 @@ const require = createRequire(import.meta.url); // eslint-disable-line
 
 const { curryN } = require("ramda");
 
-const GLOBAL_GROUP = "__global_roles__";
+const GLOBAL_GROUP = "__global_permissions__";
```

Comment:
It's not really backwards compatible if this is renamed. Is that something that matters?

Reply:
Yes, good call. I didn't really think about that, i was just renaming all `roles` to `permissions`, but i think in this case it should stay as it was.